libr/core/cmd_print.c

```diff
@@ -4354,13 +4355,56 @@ static const char* bits_to_c_code_fmtstr(int bits) {
 	}
 }
 
+static void print_c_instructions(RPrint *p, ut64 addr, const ut8 *buf, int len, int ws) {
+	size_t i;
+
+	ws = R_MAX (1, R_MIN (ws, 8));
+	int bits = ws * 8;
+	const char *fmtstr = bits_to_c_code_fmtstr (bits);
+	len /= ws;
+
+	p->cb_printf ("#define _BUFFER_SIZE %d\n", len);
```

Comment:
why do we need `cb_printf` in RCore? Can't you just use `r_cons_printf` ?

Reply:
no i cant,